src/popup/router/components/BulletSwitcher.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <div class="bullet-switcher">
+    <div class="bullet-switcher-wrapper">
+      <div
+        class="bullet-switcher-container"
+        :style="cssProps"
+      >
+        <div
+          v-for="(_, idx) in optionsSize"
+          :key="idx"
+          class="bullet"
+          :class="{
+            medium: getMediumCondition(idx),
+            big: getBigCondition(idx),
+            active: idx === currentIdx,
+          }"
+          :style="getBulletTranslation(idx)"
+          @click="() => $emit('change', idx)"
```

Comment:
```suggestion
          @click="$emit('change', idx)"
```
Why shouldn't we do it like that?